1-js/09-classes/01-class/article.md

```diff
@@ -298,54 +299,54 @@ class User {
 new User().sayHi();
 ```
 
-For a generator method, similarly, prepend it with `*`.
+Pentru o metodă generator, similar, o precedăm cu `*`.
 
-## Class properties
+## Proprietățile clasei
 
-```warn header="Old browsers may need a polyfill"
-Class-level properties are a recent addition to the language.
+```warn header="Browserele vechi pot avea nevoie de polyfill"
+Proprietațile la nivel de clasă sunt o adiție recentă.
 ```
 
-In the example above, `User` only had methods. Let's add a property:
+În exemplul de mai sus, `User` avea doar metode. Hai să adăugam o proprietate:
 
 ```js run
 class User {
 *!*
-  name = "Anonymous";
+  name = "Anonim";
 */!*
 
   sayHi() {
-    alert(`Hello, ${this.name}!`);
+    alert(`Salut, ${this.name}!`);
   }
 }
 
 new User().sayHi();
 ```
 
-The property `name` is not placed into `User.prototype`. Instead, it is created by `new` before calling constructor, it's the property of the object itself.
+Proprietatea `name` nu este introdusă în `User.prototype`. În schimb, aceasta este creată de `new` înainte de apelarea constructorului, it's the property of the object itself.
```

Comment:
Grammar suggestion : "creată de" -> "creeată de"

Reply:
There's a part that has not been translated : "it's the property of the object itself."